test/service/test_tdr.py

```diff
@@ -357,7 +359,7 @@ def _make_mock_entity_table(self,
                                                                **additional_columns),
                                         rows=rows)
 
-    _bq_types = {
+    _bq_types: Dict[type, str] = {
         str: 'STRING',
         bytes: 'BYTES',
         int: 'INTEGER',
```

Comment:
is `DATETIME` (or `TIMESTAMP`) and `NULL` intentionally missing here?  `datetime` and `None` are present within `BigQueryValue`.

Reply:
Nullable columns allow `NULL`/`None` as a value but it makes no sense to define a column with data type `NULL`.

Reply:
I intentionally omitted the time-related types because the python types are only supported as shorthand (`type=int` is translated to `type="INTEGER"`) and supporting this for non-primitive types didn't feel worth the hassle of importing them. If the inconsistency is considered a problem I'd opt to remove this dictionary altogether.

Reply:
@hannes-ucsc will probably want to weigh in on this.

Reply:
`datetime` is already being imported. And there is one reference to a TIMESTAMP column IIRC.